examples/timeWarpExposure.py

```diff
@@ -176,10 +179,10 @@ def run():
                     )
                     destExposure.setWcs(destWcs)
                     dTime, nIter, goodPix = timeWarp(destExposure, srcExposure, warpingControl)
-                    print "%4d  %5d  %8.1f  %6.1f, %6.1f  %7.1f %10s %8d %6.2f" % (
+                    print("%4d  %5d  %8.1f  %6.1f, %6.1f  %7.1f %10s %8d %6.2f" % (
```

Comment:
Switching to `format` is not required by the Python 3 migration. In fact, use of `format` at all is highly contentious. I always use `format` in new code as I grew up with Python 3.